buildscripts/hpat-conda-recipe/meta.yaml

```diff
@@ -1,12 +1,19 @@
+{% set version = "0.29.2" %}
+{% set buildnumber = 1 %}
+
 package:
   name: hpat
-  version: {{ GIT_DESCRIBE_TAG }}
+  version: {{version}}
```

Comment:
what's wrong about the git-way?

Reply:
For some reasons conda says that these vars are not defined